README.md

```diff
@@ -1,14 +1,18 @@
 # `next-video`
 
+[![npm version](https://img.shields.io/npm/v/next-video?style=flat-square&color=success)](http://npmjs.org/next-video) 
+[![size](https://img.shields.io/bundlephobia/minzip/next-video?label=size&style=flat-square)](https://bundlephobia.com/result?p=next-video) 
+
 Next video is a react component for adding video to your [next.js](https://github.com/vercel/next.js) application. It extends both the `<video>` element and your Next app with features for automatic video optimization.
 
 - **Smart storage:** Store large video files outside of your git repo
-- **Auto optimized**: Automatically optimize video files and deliver via CDN for better playback performance and quality
+- **Auto optimized**: Optimize video files and deliver via CDN for better playback performance and quality
```

Comment:
I made this shorter because this is the only bullet with an awkward newline.
feel free to give another suggestion in shortening the wording